ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/SampleData.java

```diff
@@ -614,7 +645,7 @@ public static CCDStatementOfMeans getCCDStatementOfMeans() {
     }
 
     public static CCDCase getCCDCitizenCaseWithOperationIndicators(
-                 Supplier<CCDClaimSubmissionOperationIndicators> claimIndicatorSupplier) {
+        CCDClaimSubmissionOperationIndicators claimIndicatorSupplier) {
```

Comment:
Please rename the parameter accordingly.

Reply:
👍